Makefile

```diff
@@ -0,0 +1,35 @@
+# kernel-style V=1 build verbosity
+ifeq ("$(origin V)", "command line")
+	BUILD_VERBOSE = $(V)
+endif
+
+ifeq ($(BUILD_VERBOSE),1)
+	Q =
+else
+	Q = @
+endif
+
+export GO15VENDOREXPERIMENT:=1
+
+VERSION=$(shell git describe --dirty)
+REPO=github.com/coreos/locksmith
+LD_FLAGS="-w -s -extldflags -static"
```

Comment:
We don't have other projects setting these, but looks like they're just from before